s2/cell_test.go

```diff
@@ -559,9 +558,8 @@ func TestCellContainsPoint(t *testing.T) {
 }
 
 func TestCellContainsPointConsistentWithS2CellIDFromPoint(t *testing.T) {
-	// About 1% flaky with a random seed.
-	// TODO: https://github.com/golang/geo/issues/120
-	rand.Seed(1)
+	// TODO: Is it still about 1% flaky with a random seed.
```

Comment:
Maybe even better to assume this has been fixed (you tested it); delete these comments entirely and I'll let you know if there's still a problem.